scripts/get_latest_release.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Try to get the latest release
+url=$(curl -s https://api.github.com/repos/contiv/install/releases/latest  | grep -m 1 "browser_download_url")
+
+if [[ "$?" != "0" ]]; then
+  url=$(curl -s https://api.github.com/repos/contiv/install/releases  | grep -m 1 "browser_download_url")
+  if [[ "$?" != "0" ]]; then
+  	exit 1
+  fi
+fi
+release=$(echo $url | awk -F "/" '{print $8}')
```

Comment:
Awesome!  Debian/Ubuntu and RHEL/CentOS definitely come with it, so I think we're good on that front 😄